### DISCUSSION TITLE
A "Basic" global model for MESSAGEix

### DISCUSSION THREAD

**Discussion by panday1995** [iiasa/message_ix#426] [MESSAGE modeling]
Dear developers,

Is there any 'basic' model database (containing general technology parameters, demand & GDP assumption for e.g. the world) for MESSAGE? For example, a database that contains input data to reproduce, for example, the results of IPCC AR5 scenarios by MESSAGE? (As I saw in the supplementary file of Applied Energy 216 (2018) 649–661, that the MESSAGE model contains over 100 technologies. So I guess it is nearly impossible for a single user to gather this much information from iterature)

I have read through the documentation, but only found two tutorials. In IIASA's database there are only scenario results of each model. But detailes, such as technical parameters that adopted by the model is not shown.

I am really grateful if you can provide any information or citation.

Thank you!!

**Comment by francescolovat** [comment]
Dear @panday1995,

I'm afraid I can't answer you with the point of availability of a general dataset with technology parameters.

Have you been checking [this documentation site](https://docs.messageix.org/projects/global/en/master/index.html#) specifically? It's the documentation for the MESSAGE-GLOBIOM goblal model. There are explanations about the technologies available. To complement it, I could point you to the [Global Energy Assessment](https://iiasa.ac.at/web/home/research/Flagship-Projects/Global-Energy-Assessment/Chapters_Home.en.html) publication, where you can find most of the cost assumptions in Chapters 11 and 12. However, latest updates have been based on World Energy Outlook data and can't be shared. 

In regards of the GDP, you can look and download the different projections from the [SSP Database](https://tntcat.iiasa.ac.at/SspDb/dsd?Action=htmlpage&page=welcome), logging in as a guest. You'll also find population projection and urban/rural shares, from different organizations and energy models.

I hope this is useful.

**Comment by panday1995** [comment]
Thank you so much sir!

Before, I only checked MESSAGE-ix's official doc and relavent academic publications. Haven't touched upon the MESSAGE-GLOBIOM model (Since I was thinking this is some more advanced models).

**Comment by panday1995** [comment]
For others who may interest, I just found [this paper](https://www.sciencedirect.com/science/article/pii/S0360544218325039) also very helpful.
